com.bluejeans.BlueJeans.desktop

```diff
@@ -1,10 +1,11 @@
 [Desktop Entry]
-Name=Blue Jeans
+Name=BlueJeans
```

Comment:
I believe the correct name is "Blue Jeans", but "BlueJeans" is also used sometimes.

Reply:
BlueJeans is what's used in the .desktop file that's now in the RPM, best stick with that.